lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -935,6 +936,24 @@ static Value createLinalgPayloadCalculationForElementwiseOp(
     return b.create<arith::SelectOp>(loc, pred, scalar, zero);
   }
 
+  if (auto bitwiseNot = dyn_cast<AtenBitwiseNotOp>(op)) {
+    if (bitwiseNot.getType()
+            .cast<ValueTensorType>()
+            .getDtype()
+            .isa<mlir::FloatType>()) {
+      bitwiseNot.emitError("Bitwise_Not does not support floating point dtype");
+      return nullptr;
+    }
+    Type elementType = converter->convertType(bitwiseNot.getType())
+                           .cast<RankedTensorType>()
+                           .getElementType();
```

Comment:
nit: can you move this up before the `FloatType` check? You should be able to then do `if (elementType.isa<mlir::FloatType>()) { // emit error...` instead